### DISCUSSION TITLE
Remove CircleCI?

### DISCUSSION THREAD

**Discussion by baentsch** [open-quantum-safe/liboqs#1344] [General]
This is to ask whether we want to keep using CircleCI.

It apparently has security issues (secret rotation request) and is not exactly easy to understand with regard to concurrency (looks like we cannot run several build jobs in parallel any more): 
![image](https://user-images.githubusercontent.com/57787676/210965532-3d9c81c4-e786-4e6b-8578-8815ab93d444.png)

See our apparent "Plan limit"
![image](https://user-images.githubusercontent.com/57787676/210966278-5910dfe3-6eef-4616-85aa-90252015e6cf.png)

Also, it seems they are changing the framework such that we need to re-write some tasks anyway: From a mail received two days ago:

> Do you still need to use Docker's experimental mode? Would a [bump of the underlying Docker version](https://circleci.com/docs/building-docker-images/#docker-version) to the latest version give you the Docker features you need without requiring experimental mode?

**Comment by dstebila** [comment]
CircleCI is the bulk of our CI at this point, so I think it would be quite a lot of work to move somewhere else.  I don't think we want to move back to Travis and AppVeyor is not appropriate either, so the only other option would be Github Actions.  I'd rather try to deal incrementally with the changes CircleCI is throwing our way instead of throwing it our entirely.
